tests/Routing/RoutingUrlGeneratorTest.php

```diff
@@ -128,7 +128,7 @@ public function testControllerRoutesWithADefaultNamespace()
         $route = new Illuminate\Routing\Route(['GET'], 'foo/bar', ['controller' => 'namespace\foo@bar']);
         $routes->add($route);
 
-        $route = new Illuminate\Routing\Route(['GET'], 'something/else', ['controller' => 'something\foo@bar']);
+        $route = new Illuminate\Routing\Route(['GET'], 'something/else', ['controller' => '\something\foo@bar']);
```

Comment:
The original structure is an original implementation since `v5.0.0` release

https://github.com/laravel/framework/blob/v5.0.0/src/Illuminate/Routing/UrlGenerator.php#L522-L534
https://github.com/laravel/framework/blob/v5.0.0/tests/Routing/RoutingUrlGeneratorTest.php#L118-L138